draft-ietf-rtcweb-jsep.xml

```diff
@@ -1962,9 +1962,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <t>The next step is to generate m= sections, as specified in
           <xref target="RFC4566" />, Section 5.14. An m= section is
           generated for each RtpTransceiver that has been added to the
-          PeerConnection, excluding any stopped RtpTransceivers. This
+          PeerConnection, excluding any stopped RtpTransceivers; this
           is done in the order the RtpTransceivers were added to the
-          PeerConnection.</t>
+          PeerConnection. If there are no such RtpTransceivers, no
+          m= sections are generated, as discussed in
+          <xref target="RFC3264" />, Section 5.</t>
```

Comment:
Do you want to say the semantics of this, e.g., 
"generated, allowing more streams to be added later, as discussed"

Reply:
I figured that was sort of implicit in the context of this document, but wdyt?

Reply:
It would have made it a bit easier for me to read, but I don't feel strongly about it.

Reply:
Took a shot at this, PTAL